### DISCUSSION TITLE
Dag schedule_interval change issue

### DISCUSSION THREAD

**Discussion by DanielMorales9** [apache/airflow#25304] [Q&A]
Hi everyone, 

I would like to know if it is still required in Airflow v2 to change the name of the dag when changing the `schedule_interval`.
as stated in the [Common Pittfalls](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=62694614) page.

> When needing to change your start_date and schedule interval, change the name of the dag (a.k.a. dag_id) - I follow the convention : my_dag_v1, my_dag_v2, my_dag_v3, my_dag_v4, etc...
>     Changing schedule interval always requires changing the dag_id, because previously run TaskInstances will not align with the new schedule interval

I would like to know in which version this was fixed since I could not find it in the release notes. Also I would like to know if there are any known issues related to the schedule_interval change. 

Thank you

**Comment by potiuk** [comment]
Yes. it's needed. No version, no other issues than those in common pitfalls.

**Reply by potiuk** [reply]
Watch for AIP-36 being implemented (maybe this year, maybe next) as this is the only way it can be solved. https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-36+DAG+Versioning
